docs/advanced-config.md

```diff
@@ -32,7 +32,6 @@ The following table contain the configuration options supported by this distribu
 | OTEL_PROPAGATORS                     | propagators                     | `tracecontext,baggage`                | Comma-delimited list of propagators to use. Valid keys: `baggage`, `tracecontext`, `b3multi`, `b3`.
 | OTEL_SERVICE_NAME                    | serviceName                     | `unnamed-node-service`                | The service name of this Node service.
 | SPLUNK_ACCESS_TOKEN                  | accessToken                     |                                       | The optional access token for exporting signal data directly to SignalFx API.
-| OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT | maxAttrLength                 | 1200                                  | Maximum length of string attribute value in characters. Longer values are truncated.
```

Comment:
We shouldn't remove documentation for this as we are setting this to non-otel default.

Reply:
We're setting non-otel default to `OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT`(very similar name, but more general) and that is documented :point_up: on line 23 here.